source/install/cloud-changelog.md

```diff
@@ -4,12 +4,50 @@ This changelog summarizes updates to [Mattermost Cloud](https://mattermost.com/g
 
 Latest Mattermost Cloud releases:
 
+- [Release 2023-05-18](#release-2023-05-18)
 - [Release 2023-05-01](#release-2023-05-01)
 - [Release 2023-04-21](#release-2023-04-21)
 - [Release 2023-03-29](#release-2023-03-29)
 - [Release 2023-03-20](#release-2023-03-20)
 - [Release 2023-01-26](#release-2023-01-26)
-- [Release 2023-01-16](#release-2023-01-16)
+
+## Release 2023-05-18
+
+### Improvements
+
+#### User Interface (UI)
+ - Upgraded the pre-packaged GitHub plugin version to 2.1.5.
+ - Upgraded the pre-packaged Autolink plugin version to 1.4.0.
+ - Added an experimental feature to disable re-fetching of channel and channel members on browser focus.
+ - Bot users are now hidden in the user selector in apps forms.
+ - Removed the fetching of archived channels on page load.
+ - The "Channel Type" dropdown within the "Browse Channels" modal can now be focused.
+ - Removed in-app help pages that were no longer accessible.
+ - Removed system join/leave messages from thread replies and post them instead in the main channel.
+ - Added a setting to make the channel autocomplete only appear after typing a couple letters instead of immediately after a tilde.
```

Comment:
```suggestion
 - Added an experimental setting to make the channel autocomplete only appear after typing two charactters instead of immediately after the tilde (~).
```